src/main/java/com/bandwidth/sqs/consumer/SqsConsumer.java

```diff
@@ -82,6 +84,7 @@ public SqsConsumer(SqsConsumerBuilder<T> builder) {
         this.expirationStrategy = requireNonNull(builder.expirationStrategy);
 
         this.sqsQueue = builder.sqsQueue;
+        this.queueAttributes = sqsQueue.getAttributes().blockingGet();
```

Comment:
it's a constructor. I _could_ make this non-blocking, but I don't really see the benefit here for how much code would need to change. I'll open an issue on this to remember it though